packages/odyssey-react-mui/src/Toast.tsx

```diff
@@ -70,7 +70,7 @@ export type ToastProps = {
   /**
    * The text content of the Toast
    */
-  text: string;
+  text: string | ReactNode;
```

Comment:
We should probably not allow `ReactNode` here without design approval.

Reply:
Does Odyssey have a rule or policy wrt `string` vs `ReactNode` and `text` vs `children`? I feel we can avoid churn if it's clear 
- `text` is always `string`
- `children` is always `ReactNode`

Reply:
removed for now, as it's not strictly needed for downstream apps